docs/core/project-sdk/overview.md

```diff
@@ -36,22 +36,22 @@ For Windows Forms and Windows Presentation Foundation (WPF) projects, you specif
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
-  ...
+    <!-- Omitted for brevity... -->
 </Project>
 ```
 
-Starting with .NET Aspire 9, the preceding example could instead use the .NET Aspire SDK.
+The `Project/Sdk` attribute and `Sdk` element enable additive SDKs. Consider the following example, where the .NET Aspire SDK (`Aspire.AppHost.Sdk`) is added to the project atop the `Microsoft.NET.Sdk`:
```

Comment:
Is this a conventional way to refer to an attribute? `Project/Sdk`

Reply:
Yes, I've seen it done this way many times before. A less common approach is using `.` notation, but that IMO can be misleading.